SimG4Core/Application/python/g4SimHits_cfi.py

```diff
@@ -150,6 +150,9 @@
         FlagBERT    = cms.untracked.bool(False),
         EminFTFP    = cms.double(3.), # in GeV
         EmaxBERT    = cms.double(6.), # in GeV
+        EminQGSP    = cms.double(12.), # in GeV
```

Comment:
@bsunanda , I do not in favor for addition of too many these parameter. From one hand, not a big deal_ from other ' these parameters are used a bit differently for 10.4 and 10.6